static/app/chartcuterie/performance.tsx

```diff
@@ -23,6 +23,15 @@ function modifyOptionsForSlack(options: Omit<LineChartProps, 'series'>) {
   options.legend.icon = 'none';
   options.legend.left = '25';
   options.legend.top = '20';
+  options.grid = slackChartDefaults.grid;
+
+  options.yAxis = options.yAxis || {};
+  options.yAxis.axisLabel = options.yAxis.axisLabel || {};
+  options.yAxis.axisLabel.fontFamily = DEFAULT_FONT_FAMILY;
+
+  options.xAxis = options.xAxis || {};
+  options.xAxis.axisLabel = options.xAxis.axisLabel || {};
```

Comment:
Are you doing these just to not get lint errors?

You can also do it with `&&` like
```
if (options.xAxis && options.xAxis.axisLabel) ...

Reply:
hmm, i should actually do it that way next time. ill update this when i modify the style in my next pr.